src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2845,6 +2845,27 @@ End Class
             End Using
         End Function
 
+        <WorkItem(18785, "https://github.com/dotnet/roslyn/issues/18785")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function BackspaceSoftSelectionIfNotPrefixMatch() As Task
+            Using state = TestState.CreateVisualBasicTestState(
+                            <Document><![CDATA[
+Class C
+    Sub Do()
+        Dim x = new System.Collections.Generic.List(Of String)()
+        x.$$Add("stuff")
```

Comment:
In the interest of preserving muscle memory for VB customers, we are mimicking the Dev12 (native) behavior, which actually switches to prefix matching when backspacing. Indeed, Dev12 selects `xAdd` in this scenario and we will do the same with this change.